### DISCUSSION TITLE
Bricks Payment Card Problem with theme Dark

### DISCUSSION THREAD

**Discussion by emkcloud** [mercadopago/sdk-js#195] [General]
In Dark mode input with Iframe are white color and digit number is not visible
I use npm install @mercadopago/sdk-js but with <script src="https://sdk.mercadopago.com/js/v2"></script> same problem
![image](https://github.com/mercadopago/sdk-js/assets/5731295/9d237d42-a735-4867-a5d7-af5a309c17ed)

**Comment by ghost** [comment]
Hello @massimodellarovere, we analyzed your case, but were unable to reproduce the same issue.
We recommend that you review your CSS looking for a likely style override in the `<input>` tags.
By default, all card inputs in Card Payment Brick comes with `background: transparent`, it allows us to apply the requested theme by you.

**Comment by emkcloud** [comment]
Hola Andrea, I found the problem. I have a website page with css color-scheme:dark an the result is this:
![image](https://github.com/mercadopago/sdk-js/assets/5731295/ff92b95f-49ae-4459-a3f7-e5ff582f0bc8)
But If I overlay  CSS value in container MercadoPago
![image](https://github.com/mercadopago/sdk-js/assets/5731295/f419931c-e3f5-479c-8b86-7248e862886a)
Result is OK
![image](https://github.com/mercadopago/sdk-js/assets/5731295/5df68f21-80e2-4574-9927-1b08a34cf193)

**Comment by ghost** [comment]
Nice!
I'm glad you managed to resolve it.
If you need help with anything else, please contact us.
I will close this discussion, thank you for using Mercado Pago!
